triage/update_summaries.sh

```diff
@@ -66,7 +68,7 @@ mkdir -p slices
 
 pypy summarize.py \
   triage_builds.json \
-  triage_tests.json \
+  triage_tests/* \
```

Comment:
To ensure the gzipped files do not get passed (e.g. `shard_000000000000.json.gz`)
```suggestion
  triage_tests/*.json \
```